wandb/sdk/launch/launch.py

```diff
@@ -101,7 +101,7 @@ def run(
     parameters: Parameters (dictionary) for the entry point command. Defaults to using the
         the parameters used to run the original run.
     docker_args: Arguments (dictionary) for the docker command.
-    experiment_name: Name of experiment under which to launch the run.
+    name: Name of run under which to launch the run.
```

Comment:
"name under which to launch the run" maybe

Reply:
Oops, yeah thats better